### DISCUSSION TITLE
Badge system

### DISCUSSION THREAD

**Discussion by quickgaga489** [community/community#20679] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm new here and I don't understand some things. I've read a few articles about the badge system, but there's still something I don't understand.In some example profiles, the same repo appears 3 times in the star badge, and some of them have higher levels even though they look the same as the others. Why?

**Comment by owerdogan** [comment, marked as answer]
Hello there!
Did you read this article? https://github.com/Schweinepriester/github-profile-achievements
Tiers are described in this section: https://github.com/Schweinepriester/github-profile-achievements#tiers

**Reply by quickgaga489** [reply]
I looked at similar questions but I guess I missed them, thank you
